.github/workflows/build.yml

```diff
@@ -0,0 +1,58 @@
+name: Build the Python app
+run-name: ${{ github.actor }} - ${{ github.ref_name }}
```

Comment:
The run name should include the commit hash. Consider updating it to something like `${{ github.actor }} - ${{ github.ref_name }} - ${{ github.sha }}` to include the commit hash.